Cargo.toml

```diff
@@ -1,7 +1,7 @@
 [package]
 
 name = "selectors"
-version = "0.1.1"
+version = "0.1.3"
```

Comment:
Since this PR has breaking API changes, this should have been `0.2.0` so that `cargo update` typically won’t upgrade automatically.

The default type of version requirement in Cargo is `^`, which means roughly "anything compatible with this version per Semantic Versioning, extrapolated into 0.x.y" http://doc.crates.io/crates-io.html#using-cratesio-based-crates So `cssparser = "0.1.2"` in the `[dependencies]` section of a `Cargo.toml` file is equivalent to `cssparser = ">=0.1.2, <0.2.0"`